cdap-watchdog/src/test/java/co/cask/cdap/logging/write/LogCleanupTest.java

```diff
@@ -425,7 +425,7 @@ public void testCleanFilesWithoutMeta() throws Exception {
     final Map<String, NamespaceId> namespaceIdMap = new HashMap<>();
 
 
-    // Here, we have kept disk scan limit as 15. Since we do not collect files modified after till time, we only have
+    // Here, we have kept disk scan limit as 15. Since we do not collect files modified after "till time", we only have
```

Comment:
what it really should be is
```
"untilTime"
```
because (if I understand the code correctly) it is referring to is that variable in the file at
https://github.com/caskdata/cdap/pull/7517/files/bcef04e25813c9807c56834b5d9dbec8beff400c#diff-ca501fb340ff11008c3413c36c21619bR85
which was named `tillTime`...